networks/nodle-solochain/chaintypes.ts

```diff
@@ -0,0 +1,35 @@
+export default {
```

Comment:
This type format (not bundleTypes) can meet errors when run validation on subquery, if not you can leave it like this

Reply:
works right:
<img width="1044" alt="Screenshot 2022-04-19 at 16 35 18" src="https://user-images.githubusercontent.com/40560660/164016030-621b354c-27f1-480b-b36b-03c3d1e5048a.png">